Services/SelectedStopPointManager.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace CanalTP\MttBundle\Services;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\Serializer\Serializer;
+
+use CanalTP\MttBundle\Entity\LineTimetable;
+use CanalTP\MttBundle\Entity\SelectedStopPoint;
+
+/**
+ * Class SelectedStopPointManager
+ * @package CanalTP\MttBundle\Services
+ */
+class SelectedStopPointManager
+{
+    private $om = null;
+    private $repository = null;
+    private $serializer = null;
+    private $navitia = null;
+
+    /**
+     * Constructor
+     *
+     * @param ObjectManager $om
+     */
+    public function __construct(ObjectManager $om, Navitia $navitia, Serializer $serializer)
```

Comment:
yes it breaks something
 with jms_serializer.camel_case_naming_strategy.class: JMS\Serializer\Naming\IdenticalPropertyNamingStrategy

it works but I don't know if it's the right strategy used in our others applications.
And I don't know if we really use Jms serializer.